SourceX/DiabloUI/credits.cpp

```diff
@@ -236,23 +230,22 @@ void CreditsRenderer::Render()
 	// We use unscaled coordinates for calculation throughout.
 	Sint16 dest_y = VIEWPORT.y - (offset_y - lines_begin * LINE_H);
 	for (std::size_t i = 0; i < lines_.size(); ++i, dest_y += LINE_H) {
-		auto &line = lines_[i];

```

Comment:
Does C++03 not have references?

Reply:
It doesn't have `auto`, so my guess is that this was just the quick way to solve it. I changed it to `CachedLine`